src/providers/ext.rs

```diff
@@ -0,0 +1,56 @@
+use crate::models::service::safes::AddressEx;
+use crate::providers::address_info::AddressInfo;
+use crate::providers::info::{InfoProvider, TokenInfo};
+use rocket::futures::future::OptionFuture;
+
+// Using the pattern here:
+// use rocket::futures::stream::StreamExt;
```

Comment:
Is this a todo or why is this commented?

Reply:
I left it as reference of the `TraitExt` pattern, just to show that is common. I can remove it, if you prefer.